mat64/cholesky.go

```diff
@@ -103,6 +107,23 @@ func (c *Cholesky) SetFromU(t *TriDense) {
 	c.updateCond(-1)
 }
 
+// Clone makes a copy of the input Cholesky into the receiver, overwriting the
+// previous value of the receiver. Clone does not place any restrictions on receiver
+// shape. Clone panics if the input Cholesky is not the result of a valid decomposition.
+func (c *Cholesky) Clone(chol *Cholesky) {
```

Comment:
I don't think so. This follows the spirit of the invariant: "Matrix input arguments to functions are never directly modified. If an operation changes Matrix data, the mutated matrix will be the receiver of a function."

Given this invariant, the modified `Cholesky` should be the receiver. At that point, the From in `CloneFrom` is redundant. This also matches (*Dense).Clone.